robocorp-code/src/robocorp_code/robocorp_language_server.py

```diff
@@ -1048,28 +1052,18 @@ def _remove_locator_from_json_internal(
 
         path = Path(params["robotYaml"])
         name = params["name"]
-        locators_json = path.parent / "locators.json"
-        locator: Locator
-        if locators_json.exists():
-            try:
-                db = LocatorsDatabase(str(locators_json))
-                db.load()
-                if not db.locators.items():
-                    error = db.error
-                    if not isinstance(error, str):
-                        if isinstance(error, tuple) and len(error) == 2:
-                            try:
-                                error = error[0] % error[1]
-                            except:
-                                error = str(error)
-                        else:
-                            error = str(error)
-
-                    return {"success": False, "message": error, "result": None}
+        db, locators_json = None, None
+        action_result: ActionResultDictLocatorsJson = self._load_locators_db(path)
```

Comment:
I think that this line should be inside the `try..except` below as it can raise an exception when loading the db.

Reply:
added inside try..except